packages/fields/src/types/Location/views/Field.js

```diff
@@ -8,13 +8,19 @@ import { Map, Marker, GoogleApiWrapper } from 'google-maps-react';
 
 const LocationField = ({
   field,
-  value: { googlePlaceID, formattedAddress, lat, lng } = {},
+  value = {},
```

Comment:
```suggestion
  value,
```

Default no longer needed.